tests/Unit/Middleware/CaseConversionTest.php

```diff
@@ -69,6 +69,7 @@ public function testWillRemainUntouchedOnSameReplacementKeyAsOriginalKey(): void
 
     public function conversionDataProvider(): array
     {
+        // phpcs:disable Generic.Files.LineLength
```

Comment:
Would also just introduce additional new lines. And avoid using the phpcs annotations.

```
'Studly caps to camel case' => [
    TextNotation::STUDLY_CAPS(),
    TextNotation::CAMEL_CASE(),
    'DeliveryAddress',
    'deliveryAddress'
],
...
```

Reply:
I'm not fan of this change, it's too many lines, here that compact style is better. Solutions:
1. remove lines length check for that file as I suggest in another comment below
2. that array can be generated from from little-bit changed `possibleTextNotationDataProvider` array:
```php
'Studly caps' => [
    'notation' => TextNotation::STUDLY_CAPS(),
    'property' => 'DeliveryAddress', // or examle
],
...
```

Reply:
Lets go for option 1: the custom `phpcs.xml.dist` and skip line length checking for the `tests` folder.

Reply:
replied below